routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
```

Comment:
I am reusing. This way both respond with a array so when I, for example, graph the top word it is the same as graphing some particular word. 

## why I need an array
multiple words might be the top word at any particular time so return a array of all the top words.